### PR TITLE
Reuse policy

### PR DESCRIPTION
**Pull request by snowhork**
`gcloud projects get-iam-policy` may be slow, so I want to reuse `FetchCurrentPolicy` result.

And, I have added verbose log.

### CODE REVIEW

**Comment by snowhork** [comment]
@hirosassa please review!

File: bqrole/project.go [hunk 1]
```diff
@@ -80,15 +80,14 @@ func grantProjectRole(project, user, role string) error {
 	}
 
 	cmd := fmt.Sprintf("gcloud projects add-iam-policy-binding %s --member %s --role %s", project, member, role)
-	err = exec.Command("bash", "-c", cmd).Run()
-	if err != nil {
+	if err := exec.Command("bash", "-c", cmd).Run(); err != nil {
```
Comment (hirosassa) [inline bqrole/project.go:83]: Thanks!

File: bqrole/project.go [hunk 2]
```diff
@@ -49,9 +49,14 @@ func PermitProject(role, project string, users []string, yes bool) error {
 
 	defer client.Close()
 
+	policy, err := FetchCurrentPolicy(project)
```
Comment (hirosassa) [inline bqrole/project.go:52]: 👍

**Review by hirosassa — APPROVED** [review]
Looks good and performance improved on my local mac. Thanks!

**Pull request merged**